examples/glyph-vector.py

```diff
@@ -5,6 +5,12 @@
 #  FreeType high-level python API - Copyright 2011 Nicolas P. Rougier
 #  Distributed under the terms of the new BSD license.
 #
+#  - The code is strickly speaking not optimal, as it ignores the 3rd order
```

Comment:
I agree, we're not talking about optimality. "incomplete" or "incorrect for some rare cases" would be better.